### PR TITLE
Reorganised the code and shifted common code.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
This is one of the TODO item.

### CODE REVIEW

File: tensorflow/lite/kernels/kernel_util.h [hunk 1]
```diff
@@ -84,6 +84,17 @@ inline void SetTensorToDynamic(TfLiteTensor* tensor) {
   }
 }
 
+inline TfLiteStatus SetTensorSizes(TfLiteContext* context, TfLiteTensor* tensor,
```
Comment (karimnosseir) [inline tensorflow/lite/kernels/kernel_util.h]: Declare only, and move definition in .cc file please

Thanks

Comment (amitsrivastava78) [reply tensorflow/lite/kernels/kernel_util.h]: @karimnosseir , thanks for the review, i have updated the code as per the suggestion, kindly check and approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@karimnosseir , thanks for the review, i have updated the code as per the suggestion, kindly check and approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
Thanks for the review,Closing this PR as a similar is already closed. 

Regards
Amit

**Pull request closed without merging**